src/pages/en/upgrade-astro.mdx

```diff
@@ -0,0 +1,53 @@
+---
```

Comment:
Nit: I know the old `/migrate` was in the root like this, but should this be in `guides/` now perhaps?

Reply:
That's a bigger question since probably more pages should go elsewhere!   :sweat_smile: 

Most of the stuff in this "Start Here" section is in the root. So I didn't know whether we wanted to adhere to that, or move to guides. Preference for guides? (And if so, I'll update the Netlify redirect, too.)

Reply:
I think whatever you prefer is OK! Just wanted to flag this as an opportunity to put this where we want. So if you’re happy with this in root, that’s cool.

Reply:
It feels "root"y to me, but I'll def keep this in mind before we undraft! Thanks for the attention to detail! :partying_face: